back-end/controllers/main-controller.js

```diff
@@ -0,0 +1,99 @@
+const connection = require("../db") 
+
+const addProduct= (req,res)=>{
+const {storeId,productName,productDescripition,quantityPerUnit,unitPrice,availableProduct,picture}=req.body
```

Comment:
You need to use extensions to fix your code style, in VS Code search for `prettier` and start using it to format your files, you will have to use on all files that you create/change to ensure that your code is matching the conventional code style